### DISCUSSION TITLE
来个大哥帮忙适配下Dell vostro 3690吧

### DISCUSSION THREAD

**Discussion by maodou38** [daliansky/Hackintosh#335] [General]
台式小主机  应该不复杂吧  有没有理论大神给帮忙做一个

**Comment by qibinghua** [comment]
付费求dell xps 7390 2 in 1



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2021年8月3日(星期二) 下午5:20
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [daliansky/Hackintosh] 来个大哥帮忙适配下Dell vostro 3690吧 (#335)





 
台式小主机  应该不复杂吧  有没有理论大神给帮忙做一个
 
—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
